src/building/suggested.md

```diff
@@ -39,11 +39,10 @@ you can write: <!-- date: 2022-04 --><!-- the date comment is for the edition be
     "rust-analyzer.procMacro.enable": true,
     "rust-analyzer.cargo.buildScripts.enable": true,
     "rust-analyzer.cargo.buildScripts.overrideCommand": [
-        "cargo",
+        "python3",
+        "x.py",
         "check",
-        "-p",
-        "rustc_driver",
-        "--message-format=json"
+        "--json-output"
     ],
```

Comment:
I assume RA will append to that the crate it cares about, or so? Because just running this command will check a *lot* of things^^ The old command specifically passed rustc_driver, so this one could pass "compiler/rustc_driver" or "compiler/rustc" or so?

Reply:
No, but RA just runs this command once for everything. It also avoids actually checking anything using RUSTC_WRAPPER.

Reply:
RUSTC_WRAPPER isn't supported by x.py.

Reply:
> RUSTC_WRAPPER isn't supported by x.py.

(Looks like you removed this comment but it still got sent out to everyone via notifications so people only reading the mails will be very confused... deleting comments is not a good idea)

Florian [says the opposite here](https://rust-lang.zulipchat.com/#narrow/stream/122651-general/topic/.22target.22.20directory.20in.20rustc.20tree.3F/near/288428070)